cmd/cli/flavor.go

```diff
@@ -31,10 +31,7 @@ func flavorPluginCommand(plugins func() discovery.Plugins) *cobra.Command {
 				return err
 			}
 
-			flavorPlugin, err = flavor_plugin.NewClient(endpoint.Protocol, endpoint.Address)
-			if err != nil {
-				return err
-			}
+			flavorPlugin = flavor_plugin.NewClient(endpoint.Address)
```

Comment:
I'm uneasy about half-supported features.  Assuming that #285 is O(months) out before we break ground, i find the added clarity of support and simplicity welcome in the meantime.